lib/ansible/plugins/action/pause.py

```diff
@@ -169,7 +170,8 @@ def run(self, tmp=None, task_vars=None):
                     if not seconds:
                         new_settings = termios.tcgetattr(fd)
                         new_settings[0] = new_settings[0] | termios.ICRNL
```

Comment:
It only makes sense to set this with `termios.ICANON`, so may as well put both inside the conditional.

Reply:
i think we can even remove ICANNON